### PR TITLE
Add formula for Valkyrie cask

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
All steps work, but live check fails due to the source repo having some odd naming conventions for versions ` - Version '2.5.6' differs from '2.5/v2.5.6a' retrieved by livecheck.` which I am trying to figure out, but maybe someone has an idea why…

File: Casks/valkyrie.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  url "https://github.com/NPBruce/valkyrie/releases/download/release/#{version.major_minor}/v#{version}/valkyrie-macos-#{version}.tar.gz",
+      verified: "github.com/NPBruce/valkyrie"
+  name "Valkyrie"
+  desc "GM for Fantasy Flight Board Games"
+  homepage "https://npbruce.github.io/valkyrie/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
```
Comment (neutric) [inline Casks/valkyrie.rb:13]: @miccal At least the last couple of pre-releases seem to have a letter tacked on to the end of the version number, which the regex alone filters out, so no need for `strategy :github_latest`?!

Comment (miccal) [reply Casks/valkyrie.rb:13]: Yes I saw that, but this makes sure it won't pick them up in case they change that.

Comment (neutric) [reply Casks/valkyrie.rb:13]: I thought `strategy :github_latest` is supposed to be used sparingly...

Comment (miccal) [reply Casks/valkyrie.rb:13]: Good point.

**Comment by neutric** [comment]
```
zap trash: [
  "~/Library/Application Support/CrashReporter/Valkyrie_*.plist",
  "~/Library/Preferences/unity.NA.Valkyrie.plist",
  "~/Library/Saved Application State/unity.NA.Valkyrie.savedState",
]
```

**Comment by miccal** [comment]
Thanks @ChrisChinchilla.

**Comment by ChrisChinchilla** [comment]
Sorry and thanks @miccal  and @neutric I was out most of the weekend and was planning to get to this today.

**Comment by neutric** [comment]
@ChrisChinchilla All good! Thanks for your contribution!

**Pull request merged**